src/options/options.html

```diff
@@ -59,157 +59,155 @@
 			</div>
 		</div>
 		<form>
-			<h1 data-i18n="__MSG_titleGeneral__">General</h1>
-			<ul class="mobile-incompatible">
+			<h1 class="first" data-i18n="__MSG_titleGeneral__">General</h1>
```

Comment:
Headings should be _inside_ of a section. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section:
> Each <section> should be identified, typically by including a heading (<h1>-<h6> element) as a child of the <section> element.